Assets/Scripts/NPCSrcipts/BoundedNPC.cs

```diff
@@ -0,0 +1,85 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class BoundedNPC : MonoBehaviour
+{
+    private Vector3 directionVector;
+    private Transform myTransform;
+    public float speed;
+    private Rigidbody2D myRigidBody;
+    private Animator myAnimator;
+    public Collider2D bounds;
+    // Start is called before the first frame update
+    void Start()
+    {
+        myAnimator = GetComponent<Animator>();
```

Comment:
You could either do this, or create public fields and assign values to them via the Inspector window in Unity 🤠